configure.ac

```diff
@@ -56,19 +56,6 @@ AC_CHECK_FUNCS([err errx warn warnx],, [LIBBSD_NEEDED=yes])
 # -Wno-error=pedantic from the CI.
 # See also: https://github.com/resurrecting-open-source-projects/scrot/pull/333#issuecomment-1572157050
 AC_CHECK_HEADERS([sys/queue.h],, [LIBBSD_NEEDED=yes], [ ])
-# libbsd is obligatory on systems that don't have the BSD functions we use.
-# Pass "--enable-libbsd-feature-test" to ./configure, and the configure script
-# will tell you whether the library is a dependency. This option is intended to
-# be used by package maintainers.
-AS_IF([test "x$enable_libbsd_feature_test" = "xyes"],
```

Comment:
Related to `AS_IF`:
https://github.com/resurrecting-open-source-projects/scrot/blob/1ced1b6aa9f6916d19ad0fb00841520db2a5399c/configure.ac#L50-L52

Reply:
Good catch, removed it too.